fiadoc/drivers.py

```diff
@@ -120,17 +139,23 @@ def _default_cache_dir(self) -> Path:
         return cache_dir
 
     @property
+    # TODO: cached_property?
```

Comment:
I'd say yes to that. I don't think there is any intention to have this code in long-running applications like servers. This will always be called as a script and have a short run time, so the cached property is "reset" on every new run of the script.